store.js

```diff
@@ -4,15 +4,16 @@ const StoreContext = createContext();
 
 const initialState = {
   currentListing: {
+    id: 0,
     title: "",
-    price: 0,
     description: "",
-    id: 0,
-    itemCategory: "",
-    charityCategory: "",
-    imageURL: "",
-    seller: "",
-    status: ""
+    price: 0,
+    donor: "",
+    status: "",
+    category: "",
+    charity: "",
+    image: "",
+    bids: []
```

Comment:
Will we want to hold post listings in the store or just post them to back end to get back as current listings?